packages/ckeditor5-theme-lark/theme/ckeditor5-table/tableproperties.css

```diff
@@ -18,9 +17,30 @@
 				margin-top: var(--ck-spacing-standard);
 
 				& .ck-toolbar__items > * {
-					width: 40px;
+					flex: 1;
 				}
 			}
 		}
+
+		&.ck-table-form__dimensions-row {
+			--ck-table-border-input-width: 80px;
```

Comment:
Maybe we could name it more generic? Like `--ck-table-form-default-input-width` or similar, that we can reuse across Table and/or TableCell form if needed??